cmd/analyze.go

```diff
@@ -32,15 +32,16 @@ var analyzeCmd = &cobra.Command{
 	Use:   "analyze",
 	Short: "Analyzes an image: [image]",
 	Long:  `Analyzes an image using the specifed analyzers as indicated via flags (see documentation for available ones).`,
-	Run: func(cmd *cobra.Command, args []string) {
+	Args: func(cmd *cobra.Command, args []string) error {
 		if err := validateArgs(args, checkAnalyzeArgNum, checkArgType); err != nil {
```

Comment:
Lets make an issue to replace the "arg count and type checking" to be their own flag types.